h2o-py/tests/testdir_misc/pyunit_showtime.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+import re
+import h2o
+from tests import pyunit_utils
+
+
+def test_show_time():
+
+    df = h2o.H2OFrame.from_python(
+        {"A": [1, 2, 3],
+         "B": ["a", "a", "b"],
+         "C": ["hello", "all", "world"],
+         "D": ["12MAR2015:11:00:00", "13MAR2015:12:00:00", "14MAR2015:13:00:00"]},
```

Comment:
This example is taken from pubdev-2291.
I'm not testing which date formats can we parse, but instead whether we can display them correctly in the client.